test/basic-querying.test.js

```diff
@@ -172,7 +177,7 @@ describe('basic-querying', function() {
       User.find({skip: 1, limit: 4, order: 'seq'}, function(err, users) {
         should.exists(users);
         should.not.exists(err);
-        users[0].seq.should.be.eql(1);
+        if (!isCassandraConnector) users[0].seq.should.be.eql(1);
```

Comment:
> Cassandra doesn't support absolute paging, i.e., skip or offset is not allowed

Makes sense. Let's add a new flag - `connectorCapabilities.absolutePaging` - and skip all tests that are requiring this feature (do not skip assertions only).